instrumentation/jboss-logmanager/jboss-logmanager-appender-1.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jbosslogmanager/appender/v1_1/LoggingEventMapper.java

```diff
@@ -30,7 +26,8 @@ public final class LoggingEventMapper {
 
   public static final LoggingEventMapper INSTANCE = new LoggingEventMapper();
 
-  private static final Cache<String, AttributeKey<String>> mdcAttributeKeys = Cache.bounded(100);
```

Comment:
As far as I understand this cache is used to limit creation of `AttributeKey` objects. If you believe that this cache is unnecessary then perhaps it would be better to do this in a separate PR.

Reply:
Or more specifically, the code calls `logRecordBuilder.setAttribute(String, ...)` instead of `logRecordBuilder.setAttribute(AttributeKey, ...)`

However, I just noticed that `logRecordBuilder.setAttribute(String, ...)` internally constructs an uncached `AttributeKey`.

I'll put the cache back in.

Side note, seems like `LogRecordBuilder` should cache attribute keys, so all callers of `setAttribute(String, ...)` wouldn't need to cache.  But that's definitely for another PR